package.json

```diff
@@ -20,6 +20,7 @@
     "@actions/core": "^1.11.1",
     "@actions/github": "^6.0.0",
     "@ai-sdk/anthropic": "^0.0.56",
+    "@ai-sdk/google": "^0.0.55",
```

Comment:
The `@ai-sdk/google` package is using a caret version range (`^0.0.55`). For a new integration, especially with an SDK version below 1.0.0, it's recommended to pin the exact version (`0.0.55`) to avoid potential breaking changes in patch updates. This is particularly important since 0.x versions don't follow semantic versioning strictly.